data/projects/coala_cli.md

```diff
@@ -1,24 +1,53 @@
-[cEP-0004](https://github.com/coala/cEPs/pull/29/files) was meant to improve
-the usability of coala. It is a very important proposal that never got the time
-and love it deserved.
+coala has come a long way from it's early days in terms of it's usability. But
```

Comment:
Maybe its not right to start with a conjunction here.. "usability. But" -> "usability, although"
also, "it's" - > "its"